Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/jupyter_cache/executors/utils.py", line 56, in single_nb_execution
    record_timing=False,
  File "/usr/local/lib/python3.7/dist-packages/nbclient/client.py", line 1204, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/usr/local/lib/python3.7/dist-packages/nbclient/util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/usr/local/lib/python3.7/dist-packages/nbclient/util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "/usr/lib/python3.7/asyncio/base_events.py", line 587, in run_until_complete
    return future.result()
  File "/usr/local/lib/python3.7/dist-packages/nbclient/client.py", line 664, in async_execute
    cell, index, execution_count=self.code_cells_executed + 1
  File "/usr/local/lib/python3.7/dist-packages/nbclient/client.py", line 965, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/usr/local/lib/python3.7/dist-packages/nbclient/client.py", line 862, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
data = pd.DataFrame(data,columns=xb)
------------------

[0;31m[0m
[0;31mValueError[0mTraceback (most recent call last)
[0;32m<ipython-input-17-0dad0e53d87b>[0m in [0;36m<module>[0;34m[0m
[0;32m----> 1[0;31m [0mdata[0m [0;34m=[0m [0mpd[0m[0;34m.[0m[0mDataFrame[0m[0;34m([0m[0mdata[0m[0;34m,[0m[0mcolumns[0m[0;34m=[0m[0mxb[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m
[0;32m/usr/local/lib/python3.7/dist-packages/pandas/core/frame.py[0m in [0;36m__init__[0;34m(self, data, index, columns, dtype, copy)[0m
[1;32m    607[0m         [0;32mif[0m [0misinstance[0m[0;34m([0m[0mdata[0m[0;34m,[0m [0;34m([0m[0mBlockManager[0m[0;34m,[0m [0mArrayManager[0m[0;34m)[0m[0;34m)[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m
[1;32m    608[0m             mgr = self._init_mgr(
[0;32m--> 609[0;31m                 [0mdata[0m[0;34m,[0m [0maxes[0m[0;34m=[0m[0;34m{[0m[0;34m"index"[0m[0;34m:[0m [0mindex[0m[0;34m,[0m [0;34m"columns"[0m[0;34m:[0m [0mcolumns[0m[0;34m}[0m[0;34m,[0m [0mdtype[0m[0;34m=[0m[0mdtype[0m[0;34m,[0m [0mcopy[0m[0;34m=[0m[0mcopy[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m    610[0m             )
[1;32m    611[0m [0;34m[0m[0m

[0;32m/usr/local/lib/python3.7/dist-packages/pandas/core/generic.py[0m in [0;36m_init_mgr[0;34m(cls, mgr, axes, dtype, copy)[0m
[1;32m    260[0m         [0;32mfor[0m [0ma[0m[0;34m,[0m [0maxe[0m [0;32min[0m [0maxes[0m[0;34m.[0m[0mitems[0m[0;34m([0m[0;34m)[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m
[1;32m    261[0m             [0;32mif[0m [0maxe[0m [0;32mis[0m [0;32mnot[0m [0;32mNone[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m
[0;32m--> 262[0;31m                 [0maxe[0m [0;34m=[0m [0mensure_index[0m[0;34m([0m[0maxe[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m    263[0m                 [0mbm_axis[0m [0;34m=[0m [0mcls[0m[0;34m.[0m[0m_get_block_manager_axis[0m[0;34m([0m[0ma[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[1;32m    264[0m                 [0mmgr[0m [0;34m=[0m [0mmgr[0m[0;34m.[0m[0mreindex_axis[0m[0;34m([0m[0maxe[0m[0;34m,[0m [0maxis[0m[0;34m=[0m[0mbm_axis[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m

[0;32m/usr/local/lib/python3.7/dist-packages/pandas/core/indexes/base.py[0m in [0;36mensure_index[0;34m(index_like, copy)[0m
[1;32m   6334[0m     [0;32melse[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m
[1;32m   6335[0m [0;34m[0m[0m
[0;32m-> 6336[0;31m         [0;32mreturn[0m [0mIndex[0m[0;34m([0m[0mindex_like[0m[0;34m,[0m [0mcopy[0m[0;34m=[0m[0mcopy[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m   6337[0m [0;34m[0m[0m
[1;32m   6338[0m [0;34m[0m[0m

[0;32m/usr/local/lib/python3.7/dist-packages/pandas/core/indexes/base.py[0m in [0;36m__new__[0;34m(cls, data, dtype, copy, name, tupleize_cols, **kwargs)[0m
[1;32m    467[0m [0;34m[0m[0m
[1;32m    468[0m             [0mklass[0m [0;34m=[0m [0mcls[0m[0;34m.[0m[0m_dtype_to_subclass[0m[0;34m([0m[0marr[0m[0;34m.[0m[0mdtype[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0;32m--> 469[0;31m             [0marr[0m [0;34m=[0m [0mklass[0m[0;34m.[0m[0m_ensure_array[0m[0;34m([0m[0marr[0m[0;34m,[0m [0mdtype[0m[0;34m,[0m [0mcopy[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m    470[0m             [0mdisallow_kwargs[0m[0;34m([0m[0mkwargs[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[1;32m    471[0m             [0;32mreturn[0m [0mklass[0m[0;34m.[0m[0m_simple_new[0m[0;34m([0m[0marr[0m[0;34m,[0m [0mname[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m

[0;32m/usr/local/lib/python3.7/dist-packages/pandas/core/indexes/numeric.py[0m in [0;36m_ensure_array[0;34m(cls, data, dtype, copy)[0m
[1;32m    169[0m         [0;32mif[0m [0msubarr[0m[0;34m.[0m[0mndim[0m [0;34m>[0m [0;36m1[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m
[1;32m    170[0m             [0;31m# GH#13601, GH#20285, GH#27125[0m[0;34m[0m[0;34m[0m[0;34m[0m[0m
[0;32m--> 171[0;31m             [0;32mraise[0m [0mValueError[0m[0;34m([0m[0;34m"Index data must be 1-dimensional"[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m    172[0m [0;34m[0m[0m
[1;32m    173[0m         [0msubarr[0m [0;34m=[0m [0mnp[0m[0;34m.[0m[0masarray[0m[0;34m([0m[0msubarr[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m

[0;31mValueError[0m: Index data must be 1-dimensional
ValueError: Index data must be 1-dimensional

